.github/actions/dockerhub/Dockerfile.alpine-binary

```diff
@@ -0,0 +1,22 @@
+FROM --platform=${TARGETPLATFORM} alpine as builder
+# Use a small image to download and extract the release archive
+
+ARG TAG
+ARG BIN_ARCH
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+ARG TARGETARCH
+ARG TARGETVARIANT
+ARG REPO=stacks-network/stacks-blockchain
+
+RUN case ${TARGETARCH} in \
+    "amd64")  BIN_ARCH=linux-musl-x64  ;; \
+    "arm64")  BIN_ARCH=linux-musl-arm64  ;; \
```

Comment:
Should this error out if `${TARGETARCH}` is not matched by either of these cases?  Same question for `Dockerfile.debian-binary` below.

Reply:
I hadn't considered that, since the arch is hardcoded in the workflow as a list.
I'll take a look and see what I can do about it

Reply:
added a `*) exit` statement to the `case`
https://github.com/wileyj/stacks-blockchain/commit/d8ce51e8d9a4990728bc43acf5e48c80a8079103